generate_dockerfiles.sh

```diff
@@ -58,7 +58,7 @@ for version in ${JAVA_VERSIONS[@]}; do
     DCEVM_INSTALLER_URL="https:\\/\\/github.com\\/dcevm\\/dcevm\\/releases\\/download\\/full-jdk7u79%2B8\\/DCEVM-full-7u79-installer.jar"
     DCEVM_INSTALLER_NAME="DCEVM-full-7u79-installer.jar"
   else
-    DCEVM_INSTALLER_URL="https:\\/\\/github.com\\/dcevm\\/dcevm\\/releases\\/download\\/light-jdk8u74%2B1\\/DCEVM-light-8u74-installer.jar"
+    DCEVM_INSTALLER_URL="https:\\/\\/github.com\\/dcevm\\/dcevm\\/releases\\/download\\/light-jdk8u112%2B8\\/DCEVM-light-8u112-installer.jar"
```

Comment:
as you've changed the dcevm url, it would make sense to change the local installer filename, aka DCEVM_INSTALLER_NAME below
or change it to something more generic without version in the filename